docs/dictionary/message/touchEnd.lcdoc

```diff
@@ -20,14 +20,14 @@ end touchEnd
 
 Parameters:
 pTouchID:
-A number which uniquely identifies an individual physical touch action from  
+A number which uniquely identifies an individual physical touch action from
 a sequence of touch messages 
 
 Description:
 Handle the <touchEnd> message to perform an action when the user ends a
 touch sequence.
 
-The <touchEnd> message is sent to the control which received the
+The <pTouchEnd> message is sent to the control which received the
 <touchStart> message which began the touch sequence.
 
 The <touchID> parameter is a number which uniquely identifies a sequence of
```

Comment:
@livecodesam Ah I guess you meant to change the `touchID` to `pTouchID`? Also you might need to replace the `<..>` with `*..*` to make this bold.

Reply:
This seems to be one of the entries where using `<..>` on parameters works as intended.